spec/index.html

```diff
@@ -1310,13 +1301,13 @@ <h2>Security Considerations</h2>
     security considerations will vary by domain of use.
     Security tools and protocols applicable to text
     (for example, PGP encryption, checksum validation, password-protected compression)
-    may also be used on N-Quads documents.
+    may also be used on RDF documents.
     Security/privacy protocols must be imposed which reflect the sensitivity of the embedded information.</p>
 
   <p>RDF can express data which is presented to the user, such as RDF Schema labels.
     Applications rendering strings retrieved from untrusted RDF documents,
     or using unescaped characters,
-    SHOULD prevent such strings from being used to mislead the reader,
+    SHOULD ensure that malignant strings may not be used to mislead the reader.
```

Comment:
Such "ensurance" remains impossible, whether `SHOULD` or `MUST`, and this guidance should be changed to the same wording I've suggested elsewhere. If such rewording is not acceptable by the Editors' at this point, I request that an Issue be raised for this specific warning, so that we can work toward WG consensus on what the appropriate guidance is, and apply that to all documents now containing the impossible `SHOULD` or `MUST`.